src/carts/pom.xml

```diff
@@ -27,13 +27,12 @@
         <release.version>11</release.version>
         <exec.mainClass>mushop.carts.Application</exec.mainClass>
         <micronaut.runtime>netty</micronaut.runtime>
-        <micronaut.version>3.5.3</micronaut.version>
-        <micronaut.data.version>3.5.0</micronaut.data.version>
-        <micronaut.oraclecloud.version>2.1.5</micronaut.oraclecloud.version>
+        <micronaut.version>3.6.3</micronaut.version>
+        <micronaut.data.version>3.7.3</micronaut.data.version>
         <micronaut.micrometer.version>4.3.0</micronaut.micrometer.version>
```

Comment:
Can this be removed too and use whatever version comes from the BOM?

Reply:
Removed